publish.html

```diff
@@ -0,0 +1,580 @@
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<li><p class="first">Image IDs, shown in the right pane, are used to link to images in the public group.</p>
+
+<p>These can be copied and pasted into an HTML link as described below.</p>
+
+<p>In OMERO.web click on the <span class="bold">Link</span> button and copy the the URL.</p>
+
+<p>or open the image in the full viewer and copy the URL in the browser address field</p>
+
+<p>or click the <span class="bold">Image Link</span> text and copy the URL from the pop-up.</p>
+
+<p>This URL can then be emailed or used as a permanent link to the image.</p>
+
+<h3>OMERO.insight</h3>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/publishIDInsight.jpg" class="align-left" src="images/publishIDInsight.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h3>OMERO.web</h3>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/publishIDWeb.jpg" class="align-left" src="images/publishIDWeb.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/publishLink.jpg" class="align-left" src="images/publishLink.jpg" style="width: 90%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/publishViewer.jpg" class="align-left" src="images/publishViewer.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<li><p class="first">To create an HTML link to the thumbnail or the full viewer display of the image, use the following syntax where <span class="bold">1234</span> represents the image ID:</p>
+
+<p>thumbnail (default size - 96px wide):</p>
+
+<p class="code-example">https://nightshade.openmicroscopy.org/webgateway/render_thumbnail<span class="bold">1234</span>/</p>
```

Comment:
all these urls have '/' before the ID number in  the pdf but not this web version